### PR TITLE
remove marathon json-schema

### PR DESCRIPTION
**Pull request by tamarrow**

### CODE REVIEW

File: cli/tests/integrations/test_marathon.py [hunk 1]
```diff
@@ -388,6 +386,8 @@ def test_update_invalid_request():
     assert returncode == 1
     assert stdout == b''
     stderr = stderr.decode()
+    # TODO (tamar): this becomes 'Error: App '/{' does not exist\n"'
+    # in Marathon 0.11.0
```
Comment (jsancio) [inline cli/tests/integrations/test_marathon.py]: @tamarrow What is this about?

Comment (tamarrow) [inline cli/tests/integrations/test_marathon.py]: I changed the error behavior in marathon slightly.

**Comment by jsancio** [comment]
I am merging this @tamarrow . Let me know if there is anything you want me to fix.

File: cli/tests/integrations/test_marathon.py (hunk 1 above)
Comment (jsancio) [inline cli/tests/integrations/test_marathon.py]: Let's avoid TODOs in committed code. File a Jira if we need to change this. Thanks!

**Pull request merged**